app/celery/research_mode_tasks.py

```diff
@@ -1,45 +1,43 @@
-import json
 import random
 from datetime import datetime, timedelta
 
 from flask import current_app
 from notifications_utils.s3 import s3upload
-from requests import HTTPError, RequestException, request
 
-from app import notify_celery
+from app import create_uuid, notify_celery
+from app.aws.mocks import (
+    ses_hard_bounce_callback,
+    ses_notification_callback,
+    ses_soft_bounce_callback,
+    sns_failed_callback,
+    sns_s3_callback,
+    sns_success_callback,
+)
 from app.aws.s3 import file_exists
 from app.celery.process_ses_receipts_tasks import process_ses_results
+from app.celery.process_sns_receipts_tasks import process_sns_results
 from app.config import QueueNames
-from app.models import SMS_TYPE
 
-temp_fail = "7700900003"
-perm_fail = "7700900002"
-delivered = "7700900001"
+last_digit_temp_fail = "3"
+last_digit_perm_fail = "2"
+last_digit_delivered = "1"
```

Comment:
We need to change the documentation after doing this change I guess. Or use full numbers in a follow up PR

Reply:
I am supposed to sync with Bryan on this topic. There might be more changes coming up after a merge of this PR.